src/Sfa.Roatp.Register.sln

```diff
@@ -29,6 +29,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sfa.Roatp.Register.Infrastr
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "sfa.Roatp.Register.IntegrationTests", "sfa.Roatp.Register.IntegrationTests\sfa.Roatp.Register.IntegrationTests.csproj", "{7846C93F-26AE-437E-A508-2555159A92D5}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "sfa.Roatp.Register.ApiSystemTests", "sfa.Roatp.Register.ApiIntegrationTests\sfa.Roatp.Register.ApiSystemTests.csproj", "{15B0264B-3AA5-4F81-B070-A5D7786B8805}"
```

Comment:
I realise it's nit picky but could you change sfa.Roatp to Esfa.Roatp on the Api tests and the integration tests please.

The idea is that over time we'll migrate to the Esfa namespace so if it's a new project we should use that